home_work1/src/test/java/aqa_hw_1/task_1/PutPetTests.java

```diff
@@ -0,0 +1,61 @@
+package aqa_hw_1.task_1;
+
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static io.restassured.RestAssured.given;
+
+
+public class PutPetTests {
+
+    @Test
+
+    public void verifyPetCanBeModified() {
+
+        CategoryDto category = new CategoryDto(2566, "Cat");
+        PetDto pet = new PetDto(2566, category, "Tom", "available");
+
+        Response createResponse = given()
+                .contentType(ContentType.JSON)
+                .body(pet)
+                .when()
+                .post("https://petstore.swagger.io/v2/pet");
+
+
+        Response getResponse = given().get("https://petstore.swagger.io/v2/pet/2566");
+
+        Assert.assertEquals("Tom", getResponse.jsonPath().getString("name"));
+        Assert.assertEquals("available", getResponse.jsonPath().getString("status"));
+        Assert.assertEquals("Cat", getResponse.jsonPath().getString("category.name"));
```

Comment:
"Tom", "available", "Cat" should be as a separate variables in the beginning of the test and reuse them for object creation and assertions